conans/test/command/inspect_test.py

```diff
@@ -179,4 +188,177 @@ def build(self):
 exports_sources: None
 short_paths: False
 apply_env: True
-build_policy: None""", client.out)
+build_policy: None
+settings: ('os', 'arch', 'build_type', 'compiler')
+options:
+    bar: [True, False]
+    foo: [True, False]
+default_options:
+    bar: False
+    foo: True
+""", client.out)
+
+    def test_default_options_list(self):
+        client = TestClient()
+        conanfile = r"""from conans import ConanFile
+class OpenSSLConan(ConanFile):
+    name = "OpenSSL"
+    version = "1.0.2o"
+    settings = "os", "compiler", "arch", "build_type"
+    url = "http://github.com/lasote/conan-openssl"
+    license = "The current OpenSSL licence is an 'Apache style' license: https://www.openssl.org/source/license.html"
+    description = "OpenSSL is an open source project that provides a robust, commercial-grade, and full-featured " \
+                  "toolkit for the Transport Layer Security (TLS) and Secure Sockets Layer (SSL) protocols"
+    options = {"no_threads": [True, False],
+               "no_zlib": [True, False],
+               "shared": [True, False],
+               "no_asm": [True, False],
+               "386": [True, False],
+               "no_sse2": [True, False],
+               "no_bf": [True, False],
+               "no_cast": [True, False],
```

Comment:
This test can be simplified, it doesn't add much that it has so many options

Reply:
removed